src/common/admin_socket.cc

```diff
@@ -401,7 +401,7 @@ int AdminSocket::execute_command(const std::string& cmd, ceph::bufferlist& out)
   }
   string args;
   if (match != cmd) {
-    args = cmd.substr(match.length() + 1);
+    args = cmd.substr(match.length() + 1); /// RRR don't understand this line
```

Comment:
Looks to me like it's taking the portions of the matched command past the end of the matched prefix and interpreting it as arguments.

Reply:
Thanks. I know that's the intention, but the match.length(), I think, does not count the
'{prefix,' that is still in cmd

Reply:
Doesn't matter. Someone has already fixed that code.